include/spirv/unified1/spirv.core.grammar.json

```
@@ -14260,6 +14320,11 @@
           "extensions" : [ "SPV_INTEL_split_barrier" ],
           "version" : "None"
         },
+        {
+          "enumerant" : "FPGAArgumentInterfacesINTEL",
+          "value" : 6174,
+          "version" : "None"
```

The new capability should be guarded by the name of the extension.  Please check that the extension name is correct below - it is SPV_INTEL_argument_interfaces in the PR title but I think the correct name of the extension is SPV_INTEL_fpga_argument_interfaces?

Note that only the capability should be guarded by the extension, and the new decorations should only be guarded by the new capability.

```suggestion
          "enumerant" : "FPGAArgumentInterfacesINTEL",
          "value" : 6174,
          "extensions" : [ "SPV_INTEL_fpga_argument_interfaces" ],          
          "version" : "None"
```